### PR TITLE
feat(extension): save changes on panel view state change

### PR DESCRIPTION
**Pull request by pinussilvestrus**
If `files.autoSave` is configured in the VS Code environment, we will adapt the behavior to save file changes in the BPMN Editor, e.g. when the focus of the editor got lost.

Related to #71

Note: tests are currently disabled due to missing chance to configure test workspace settings (cf. [comment](https://github.com/bpmn-io/vs-code-bpmn-io/compare/71-auto-save?expand=1#diff-45327f86d4438556066de133327f4ca2R139))

### CODE REVIEW

File: src/features/editing/bpmnModelerBuilder.ts [hunk 1]
```diff
@@ -103,6 +112,7 @@ export class BpmnModelerBuilder {
           }
 
           async function saveChanges() {
+            console.log('here');
```
Comment (nikku) [inline src/features/editing/bpmnModelerBuilder.ts:115]: Should we get rid of this one?

File: src/features/editing/bpmnModelerBuilder.ts [hunk 2]
```diff
@@ -57,10 +57,19 @@ export class BpmnModelerBuilder {
 
           const vscode = acquireVsCodeApi();
 
-          // persisting
+          // (1) persit web view state
```
Comment (nikku) [inline src/features/editing/bpmnModelerBuilder.ts:60]:
```suggestion
          // (1) persist web view state
```

File: src/features/editing/bpmnModelerBuilder.ts (hunk 1 above)
Comment (nikku) [inline src/features/editing/bpmnModelerBuilder.ts:115]:
```suggestion
```

**Comment by pinussilvestrus** [comment]
Woopsie, thank you!

**Pull request merged**